workflow/artifacts/oss/oss.go

```diff
@@ -339,7 +341,51 @@ func isTransientOSSErr(err error) bool {
 
 func putFile(bucket *oss.Bucket, objectName, path string) error {
 	log.Debugf("putFile from %s to %s", path, objectName)
-	return bucket.PutObjectFromFile(objectName, path)
+	fStat, err := os.Stat(path)
+	if err != nil {
+		return err
+	}
+	// Determine upload method based on file size.
+	if fStat.Size() <= maxObjectSize {
+		log.Info("OSS Simple Uploading")
```

Comment:
Can change simple as a specific level  https://kubernetes.io/docs/contribute/style/style-guide/#avoid-words-that-assume-a-specific-level-of-understanding

Reply:
Hi, I referred to the OSS public documentation and use SimpleUpload for consistency, plz see: https://www.alibabacloud.com/help/en/oss/user-guide/simple-upload?spm=a2c63.p38356.0.0.2c072398fh5k3W#33e74f4083r6t

Reply:
In this case this a name or term, so it is appropriate